etc/security/mh_default_org.xml

```diff
@@ -167,10 +167,6 @@
     <sec:intercept-url pattern="/api/events" method="GET" access="ROLE_ADMIN, ROLE_API_EVENTS_VIEW"/>
     <sec:intercept-url pattern="/api/events/*" method="GET" access="ROLE_ADMIN, ROLE_API_EVENTS_VIEW"/>
     <sec:intercept-url pattern="/api/events/*/acl" method="GET" access="ROLE_ADMIN, ROLE_API_EVENTS_ACL_VIEW"/>
-    <sec:intercept-url pattern="/api/events/*/attachments" method="GET" access="ROLE_ADMIN, ROLE_API_EVENTS_ATTACHMENTS_VIEW"/>
-    <sec:intercept-url pattern="/api/events/*/attachments/*" method="GET" access="ROLE_ADMIN, ROLE_API_EVENTS_ATTACHMENTS_VIEW"/>
-    <sec:intercept-url pattern="/api/events/*/comments" method="GET" access="ROLE_ADMIN, ROLE_API_EVENTS_COMMENTS_VIEW"/>
-    <sec:intercept-url pattern="/api/events/*/comments/*" method="GET" access="ROLE_ADMIN, ROLE_API_EVENTS_COMMENTS_VIEW"/>
```

Comment:
ROLE_API_EVENTS_COMMENTS_VIEW (at least) is still present in `docs/guides/developer/docs/api/groups-api.md`.  I haven't searched the others, but I would imagine this is true of some of them.

Reply:
Fixed.

I've searched the entire codebase for occurrences of the removed roles now. The one you have found was the only one.